client/cypress/e2e/projects.cy.ts

```diff
@@ -10,14 +10,24 @@ import {
 } from 'cypress/utils/e2e';
 import { getNamesOfProjects } from 'cypress/utils/projects';
 import viewports from 'cypress/utils/viewports';
-import { HAS_ONBOARDING_BEEN_CLOSED, IS_ONBOARDING_DONE } from 'src/constants/localStorageKeys';
+import {
+  HAS_ONBOARDING_BEEN_CLOSED,
+  IS_CRYPTO_MAIN_VALUE_DISPLAY,
+  IS_ONBOARDING_DONE,
+} from 'src/constants/localStorageKeys';
 import getMilestones from 'src/constants/milestones';
 import { ROOT_ROUTES } from 'src/routes/RootRoutes/routes';
 
 import Chainable = Cypress.Chainable;
 
 chai.use(chaiColors);
 
+const changeMainValueToFiat = () => {
+  cy.get('[data-test=Navbar__Button--Settings]').click();
```

Comment:
Please use `navigateWithCheck` util. This function is shared in all tests. Please move it to utils.